dfastmi/batch/core.py

```diff
@@ -707,6 +708,7 @@ def write_report(
     tstag: float,
```

Comment:
If this is a legacy method move to legacy.py. Do not do this in this issue (not in scope) but create a new issue to cleanup this file for improved maintenance.

Reply:
New issue created DFAST-237.